templates/shared/_card--masterclass.html

```diff
@@ -1,11 +1,11 @@
-<div class="p-card u-no-padding col-4">
+<div class="p-card u-no-padding col-3">
   {% if session.Link %}
   <a href="{{ session.Recording }}" aria-hidden="true" tabindex="-1">
     <img class="p-card__image" alt="{{ session.Topic }} video" src="https://lh3.google.com/u/0/d/{{ session.Link }}=k">
   </a>
   <div class="p-card__inner">
-    <h3><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
-    <p class="u-text--muted">Duration: {{ session.Duration }}</p>
+    <h5 class="u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a></h5>
```

Comment:
There's a jump from heading level 2 to 5 here
```suggestion
    <h3 class="p-heading--5 u-no-margin--bottom"><a href="{{ session.Recording }}">{{ session.Topic }}</a></h3>
```